advanced/accessibility.md

```diff
@@ -11,27 +11,36 @@ keywords: accessibility wai aria jaws nvda
 
 | Alt Key   | Result      |
 |-----------|-------------|
-| Alt + F9  | Focus/jump to menu bar |
-| Alt + F10 | Focus/jump to toolbar |
-| Alt + F11 | Focus/jump to element path |
+| Alt+F9  | Focus/jump to menu bar |
+| Alt+F10 | Focus/jump to toolbar |
+| Alt+F11 | Focus/jump to element path |
 | Esc       | Close menu/submenu/dialog (also gets you back to editor area) |
-| Tab/Arrow | Navigate left/right through menu/toolbar |
+| Tab or Arrow | Navigate left/right through menu/toolbar |
 
+For additional navigation keyboard shortcuts and shortcuts for applying some commonly-used formats, see: [Keyboard shortcuts]({{site.baseurl}}/advanced/keyboard-shortcuts/).
 
 ## How to work with the editor
 
-The content area has initial focus when you navigate into an editor instance. `Up Arrow` and `Down Arrow` gives you paragraphs,<!-- REQUIRES DEV REVIEW --> headings and other items such as links. Note that NVDA and Firefox treat links as embedded characters, you have actually to navigate into them to read their texts, but they are accessible.
+When navigating to a TinyMCE editor, the keyboard focus (cursor) will be in the content area. The up and down arrow keys navigate between the lines of the editor, such as within and between paragraphs, headings and other items such as links.
 
-Press `Alt+F9` to move focus to the menu bar. `Left Arrow` and `Right Arrow` navigates through the top menu items. `Down Arrow` opens the menu. Press `Esc` once to go back to the top level. `Right Arrow` on a submenu navigate into them while `Esc` backs out.
+### Navigating the menu bar
 
-Alternatively, press `Alt+F10` to focus on the toolbars. The first toolbar has focus and you can `Tab` or arrow through the items<!-- DEV should items be buttons -->. Submenu items and panels are opened with `Down Arrow` and closed with `Esc`. When you reach the end of one toolbar the `Right Arrow` and `Left Arrow` wrap to the next and previous toolbars respectively. `Esc` brings you back into the central content area.
+To focus on the editor menu bar, press **Alt+F9**. The **Left Arrow** and **Right Arrow** keys navigate through the top menu items. The **Down Arrow** opens the highlighted menu and moves focus to the first menu item. To activate or select the highlighted menu item, use the **Enter**, **Return**, or **Spacebar**. Press the **Esc** key to collapse the menu. The **Right Arrow** key opens submenus and the **Esc** key collapses submenus. To return focus to the content area from the menu bar, use the **Esc** key.
 
-Dialogs such as Insert/Edit or Image/Video are opened from either the menus or the toolbars. These are multi-page dialogs with at least two tabs at the top. The `Right Arrow` and `Left Arrow` moves between the tabs in the tab list. `Space` makes the current tab active. Tab moves into the panel and through the controls. `Save` accepts changes, `Esc` or `Cancel` discards changes.
+### Navigating the toolbar
```

Comment:
This might not apply here, but my thesis supervisor told me I should never have a `h4` immediately after a `h3` without some form of text in-between. Feel free to ignore if that isn't relevant here

Reply:
you're thesis supervisor was correct. However in this instance I have nothing useful to add there so I'm going to opt for the minimalist approach and not add fluff here.